modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -98,16 +99,26 @@ a| `provisioningNetworkInterface` |  | The name of the network interface on node
 
 | `defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration.
 
-| `apiVIP` | | (Optional) The virtual IP address for Kubernetes API communication.
+| `apiVIPs` | a| (Optional) The virtual IP address for Kubernetes API communication.
+
+This setting must either be provided in the `install-config.yaml` file as a reserved IP from the MachineNetwork or pre-configured in the DNS so that the default name resolves correctly. Use the virtual IP address and not the FQDN when adding a value to the `apiVIPs` configuration setting in the `install-config.yaml` file. The IP address must be from the primary IPv4 or IPv6 network when using dual-stack networking. If not set, the installation program uses `api.<cluster_name>.<base_domain>` to derive the IP address from the DNS.
```

Comment:
"The IP address must be from the primary IPv4 or IPv6 network when using dual-stack networking."

In dual stack, the primary network must be ipv4. The requirements for ip version in the new config are:

ipv4 single stack: one ipv4 address
ipv6 single stack: one ipv6 address
dual stack: one ipv4 and optionally one ipv6 address, in that order